ibis_ml/steps/_encode.py

```diff
@@ -162,13 +162,13 @@ def transform_table(self, table: ir.Table) -> ir.Table:
         ).drop(*self.categories_)
 
 
-class CategoricalEncode(Step):
-    """A step for categorical encoding select columns.
+class OrdinalEncode(Step):
+    """A step for Ordinal encoding select columns.
```

Comment:
```suggestion
    """A step for ordinal encoding select columns.
```

Reply:
Actually, sklearn description is along the lines of:

```suggestion
    """A step for encoding select columns as integer arrays.
```

Do you think this would be more clear to the user, or just "ordinal encoding select columns" is good enough?